server/src/api/users/users.controller.ts

```diff
@@ -150,6 +150,19 @@ export const getUserRoles: RouteHandlerFnc = async (ctx) => {
   ctx.body = roles;
 };
 
+export const getInvitation: RouteHandlerFnc = async (ctx) => {
+  if (!ctx.state.user) throw new Error('User is required');
+
+  const invitation = await Invitation.query()
+    .where({ id: ctx.params.invitationId })
+    .withGraphFetched('roadmap')
+    .first();
```

Comment:
```suggestion
    .findById(ctx.params.invitationId)
    .withGraphFetched('roadmap');
```

`findById` should work fine for this